<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorRegisterWindow</class>
 <widget class="QMainWindow" name="DoctorRegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-90</y>
      <width>1101</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{
	border-image: url(:/IMG_2027.JPG);
}</string>
    </property>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>110</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(244,244,244,2);
color: rgb(255, 255, 255);
font: 10pt &quot;黑体&quot;;


</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>160</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(244,244,244,2);
color: rgb(255, 255, 255);
font: 10pt &quot;黑体&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="telphoneLineEdit">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>210</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(244,244,244,2);
color: rgb(255, 255, 255);
font: 10pt &quot;黑体&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="realNameLineEdit">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>260</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(244,244,244,2);
color: rgb(255, 255, 255);
font: 10pt &quot;黑体&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IDLineEdit">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>310</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(244,244,244,2);
color: rgb(255, 255, 255);
font: 10pt &quot;黑体&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="workTimeLineEdit">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>360</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(244,244,244,2);
color: rgb(255, 255, 255);
font: 10pt &quot;黑体&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="majorLineEdit">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>420</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(244,244,244,2);
color: rgb(255, 255, 255);
font: 10pt &quot;黑体&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="educationLineEdit">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>470</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(244,244,244,2);
color: rgb(255, 255, 255);
font: 10pt &quot;黑体&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerPushButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>580</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 170, 127);</string>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="educationPushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>470</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">bcakground-color: rgb(255,132,139,0);

font: 12pt &quot;04b_21&quot;;
border-radius:3px;
color: rgb(255, 255, 255);
	</string>
     </property>
     <property name="text">
      <string>学历</string>
     </property>
    </widget>
    <widget class="QPushButton" name="majorPushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">bcakground-color: rgb(255,132,139,0);

font: 12pt &quot;04b_21&quot;;
border-radius:3px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>专业</string>
     </property>
    </widget>
    <widget class="QPushButton" name="wordTimePushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>350</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">bcakground-color: rgb(255,132,139,0);

font: 12pt &quot;04b_21&quot;;
border-radius:3px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>工作时间</string>
     </property>
    </widget>
    <widget class="QPushButton" name="IDPushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">bcakground-color: rgb(255,132,139,0);

font: 12pt &quot;04b_21&quot;;
border-radius:3px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>身份证号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="realNamePushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>260</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">bcakground-color: rgb(255,132,139,0);

font: 12pt &quot;04b_21&quot;;
border-radius:3px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>真实姓名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="telphonePushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">bcakground-color: rgb(255,132,139,0);

font: 12pt &quot;04b_21&quot;;
border-radius:3px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>电话号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="passwordPushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">bcakground-color: rgb(255,132,139,0);

font: 12pt &quot;04b_21&quot;;
border-radius:3px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="usernamePushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">bcakground-color: rgb(255,132,139,0);

font: 12pt &quot;04b_21&quot;;
border-radius:3px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="picturePushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 170, 127);</string>
     </property>
     <property name="text">
      <string>选择图片</string>
     </property>
    </widget>
    <widget class="DisplayWindow" name="displayWidget" native="true">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>291</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border:1px solid #e1e0e0;
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>441</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;04b_21&quot;;</string>
     </property>
     <property name="title">
      <string>简介</string>
     </property>
     <widget class="QTextEdit" name="briefIntroductionTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>421</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(244,244,244,2);
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayWindow</class>
   <extends>QWidget</extends>
   <header>displaywindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
